common/src/main/java/com/microsoft/hydralab/common/entity/agent/EnvCapability.java

```diff
@@ -1,11 +1,15 @@
-package com.microsoft.hydralab.agent.environment;
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package com.microsoft.hydralab.common.entity.agent;
 
 import lombok.Getter;
 
 import java.io.File;
 
 @Getter
 public class EnvCapability {
+
     @SuppressWarnings("checkstyle:VisibilityModifier")
     public enum CapabilityKeyword {
         adb("--version"),
```

Comment:
Please add Xcode and iTunes into the capability list, and verify the Xcode on Mac agent and check iTunes on Windows agent.

Reply:
Will do this in another PR.